### PR TITLE
Adds axe checker to 1995 e2e tests

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1166

Branched from edu-1995-e2e-jsb, requesting to be merged back into it (not master).

Not sure that this is the "right" way to do this, but basically all I did was copy the all-fields e2e test and added `.axeCheck('main')` before every continue button click as a sort of sanity check. Is there any more to it than that? Seems too easy--like I'm missing something.

In other news, I discovered broken tests on the direct deposit page. Because of the changes to that page, we don't seem to be getting routing and account numbers, so we have some extraneous tests that I commented out (for now).

### CODE REVIEW

File: test/accessibility/edu-1995-508.spec.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        .axeCheck('.main')
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/dependents');
+
+      // Direct deposit page
+
+      // No longer works with the updated direct deposit page
+      // client
+      //   .waitForElementVisible('label[for="root_bankAccountChange"]', Timeouts.slow);
+      // EduHelpers.completeDirectDeposit(client, EduHelpers.testData.directDeposit.data, false);
```
Comment (jbalboni) [inline test/accessibility/edu-1995-508.spec.js]: I uncommented this and it worked fine. You might need to run `npm run build` again.

**Review by jbalboni — APPROVED** [review]
LGTM. The direct deposit page should be working, though.

File: test/accessibility/edu-1995-508.spec.js (hunk 1 above)
Comment (cvalarida) [inline test/accessibility/edu-1995-508.spec.js]: Odd. I'll try again, but I had just built it.

Comment (cvalarida) [inline test/accessibility/edu-1995-508.spec.js]: I get the following errors when that particular test is run:

```
_________________________________________________

 TEST FAILURE: 2 errors during execution,  0 assertions failed, 57 passed. (1m 19s)

 ✖ accessibility/edu-1995-508.spec

   - Begin application (1m 13s)

   ERROR: Unable to locate element: "input[name="root_bankAccount_accountNumber"]" using: css selector
       at Object.completeDirectDeposit (/path/to/vets-website/test/util/edu-1995-helpers.js:186:8)
       at /path/to/vets-website/test/accessibility/edu-1995-508.spec.js:102:18
       at Object.BeginApplication [as Begin application] (/path/to/vets-website/test/util/e2e-helpers.js:100:7)


   ERROR: Unable to locate element: "input[name="root_bankAccount_routingNumber"]" using: css selector
       at Object.completeDirectDeposit (/path/to/vets-website/test/util/edu-1995-helpers.js:187:8)
       at /path/to/vets-website/test/accessibility/edu-1995-508.spec.js:102:18
       at Object.BeginApplication [as Begin application] (/path/to/vets-website/test/util/e2e-helpers.js:100:7)
```

Comment (cvalarida) [inline test/accessibility/edu-1995-508.spec.js]: For the record, @jbalboni recommended offline that I run `npm install` again. That cleared it up, so I uncommented those lines.

**Pull request merged**